### DISCUSSION TITLE
Thinking about new feature I could help with, but not sure how to do it

### DISCUSSION THREAD

**Discussion by AlexanderCova** [israel-dryer/ttkbootstrap#389] [Ideas]
So was thinking about adding a boot style option for border radius, not sure if its entirely possible with Tkinter, but could be nice if implemented properly, but also not sure how to use git, so would love recommendations to help out!

**Comment by MrRukundo** [comment]
Customtkinter (ctk) has corner radius and other cool features, I use both ttk and Ctk to get the best of both. ttk is still better in so many ways thought
